gradle.properties

```diff
@@ -5,6 +5,4 @@ org.gradle.caching=true
 android.useAndroidX=true
 android.testConfig.useRelativePath=true
 android.enableJetifier=true
-android.enableD8=true
```

Comment:
@Unpublished please add a note here why you removed these

Reply:
```
WARNING:The option 'android.enableD8' is deprecated.
The current default is 'true'.
It was removed in version 7.0 of the Android Gradle plugin.
For more details, see https://d.android.com/r/studio-ui/d8-overview.html.
WARNING:The option 'android.enableR8' is deprecated.
It was removed in version 7.0 of the Android Gradle plugin.
Please remove it from `gradle.properties`.
```
Per above these options don't have any effect since they're defaults now.